lib/AppInfo/Application.php

```diff
@@ -76,7 +76,10 @@ public function registerEventsScripts(IEventDispatcher $dispatcher) {
 		// 	\OCP\Util::addScript(self::APP_ID, 'dist/collaboration');
 		// });
 		//\OCP\Util::addScript(self::APP_ID, 'beeswarmfilelist' );
+<<<<<<< HEAD
```

Comment:
There is a merge conflict here that wasn't handled properly

Reply:
Corrected